### PR TITLE
Rename deployTarget to addToModel

### PR DESCRIPTION
**Pull request by huwshimi**
Fixes: #3388.

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- check in a few places that adding to the canvas still works e.g. from the profile, search results and entity details

**Comment by huwshimi** [comment]
[Rename deployTarget](https://trello.com/c/m9qwcGkP/167-rename-deploytarget)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8386/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8387/

**Comment by mitechie** [comment]
Does this break the deployTarget url param? If so I think we need at least some code to accept that and map it to the new addToModel so that we don't break existing links out there on the internet that leverage the deployTarget url flag.

File: jujugui/static/gui/src/app/init.js [hunk 1]
```diff
@@ -1488,7 +1488,7 @@ class GUIApp {
         deployTarget: null
```
Comment (hatched) [inline jujugui/static/gui/src/app/init.js]: So this branch updates the method but leaves the state as deployTarget? Are you planning on doing a follow-up to update state and then keep support for the old/new url scheme?

**Comment by huwshimi** [comment]
@mitechie @hatched this just updates the internal method name, but leaves the URL/functionality for `deployTarget`. It separates the method that actually adds entities to the model from the name of the functionality which we use elsewhere.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8447/

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**